### PR TITLE
add unicode data to tests

### LINKED ISSUE #395: CSV export fails
CSV export fails with the following stacktrace:
```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python2.7/dist-packages/flask_appbuilder/security/decorators.py", line 26, in wraps
    return f(self, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/caravel/models.py", line 1164, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/caravel/views.py", line 515, in explore
    payload = obj.get_csv()
  File "/usr/local/lib/python2.7/dist-packages/caravel/viz.py", line 282, in get_csv
    return df.to_csv(index=include_index)
  File "/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py", line 1343, in to_csv
    formatter.save()
  File "/usr/local/lib/python2.7/dist-packages/pandas/core/format.py", line 1550, in save
    self._save()
  File "/usr/local/lib/python2.7/dist-packages/pandas/core/format.py", line 1651, in _save
    self._save_chunk(start_i, end_i)
  File "/usr/local/lib/python2.7/dist-packages/pandas/core/format.py", line 1677, in _save_chunk
    lib.write_csv_rows(self.data, ix, self.nlevels, self.cols, self.writer)
  File "pandas/lib.pyx", line 1078, in pandas.lib.write_csv_rows (pandas/lib.c:19848)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xfc' in position 1: ordinal not in range(128)
```

### PR DESCRIPTION
**Pull request by andrewhn**
As promised, this PR:
- adds a csv with unicode data for testing to `caravel/data`
- adds a loading function for the data, activated with command line flag `--test` (and by the tests)
- adds a table, slice (pic below), and dashboard featuring unicode data (if `--test` is specified)

Note I changed the `sample` flag in the data load command in `bin/caravel`, as it didn't seem to be doing anything.

Also fixes #395.

image

### CODE REVIEW

**Comment by mistercrunch** [comment]
Awesome! `--sample` was leftover from earlier logic I had to load samples faster to accelerate the tests.

File: caravel/bin/caravel [hunk 1]
```diff
@@ -68,9 +68,9 @@ def version():
     print(s)
 
 @manager.option(
-    '-s', '--sample', action='store_true',
-    help="Only load 1000 rows (faster, used for testing)")
-def load_examples(sample):
+    '-s', '--test', action='store_true',
```
Comment (mistercrunch) [inline caravel/bin/caravel]: `'-t', '--include-test-data', action='store_true',`

File: caravel/bin/caravel [hunk 2]
```diff
@@ -68,9 +68,9 @@ def version():
...
+    help="Load additional test data")
+def load_examples(test):
```
Comment (mistercrunch) [inline caravel/bin/caravel]: `def load_examples(include_test_data)`

**Comment by mistercrunch** [comment]
Looks perfect except for the few nits that I pointed out. Thanks for doing this!

**Comment by andrewhn** [comment]
No worries, should be all good :+1:

**Comment by mistercrunch** [comment]
img

**Pull request merged**